event-dispatcher/src/main/java/io/sk8s/event/dispatcher/EventDispatcherProperties.java

```diff
@@ -24,18 +24,8 @@
 @ConfigurationProperties(prefix = "sk8s.event.dispatcher")
 public class EventDispatcherProperties {
 
-	private String namespace = "default";
```

Comment:
I believe @freynca is depending on the namespace being configurable.

Reply:
The namespace should absolutely be configurable

Reply:
this is what @ericbottard changed (namespace of the event-dispatcher is inherited by all of its deployments):
https://github.com/markfisher/sk8s/pull/41/files#diff-02a72fcc3a08713e28a5ab7f8d510f71R76